frontend/components/Home.jsx

```diff
@@ -91,9 +94,12 @@ function Home__RenderFunc(props,ref) {
 					data-plasmic-for-node={forNode}
 					className={classNames(
 						"plasmic_default__all",
-						"plasmic_default__div",
-						"root_reset_mXv5TZ5SUPGRneH9RoMn6q",
-						"Home__root__rfWi8"
+                        "plasmic_default__div",
```

Comment:
Looks like tabs changed to spaces here. But we can fix the formatting later

Reply:
Oh yeah I think it's because I copy pasted it.